components/scream/src/physics/shoc/atmosphere_macrophysics.cpp

```diff
@@ -35,19 +35,22 @@ void SHOCMacrophysics::set_grids(const std::shared_ptr<const GridsManager> grids
 
   // Layout for 2D (1d horiz X 1d vertical) variable
   FieldLayout scalar2d_layout_col{ {COL}, {m_num_cols} };
-  FieldLayout scalar1d_layout_lev{ {LEV},  {m_num_levs} };
+  FieldLayout scalar2d_layout_lev{ {LEV},  {m_num_levs} };
```

Comment:
This is a bit nitpicky, but wouldn't vertical level be considered a 1d scalar variable?

Reply:
ahh that's why I had it the other way. I'll change back.